README.md

```diff
@@ -189,3 +259,6 @@ If you're implementing a complex workflow, you may want to know if you've alread
 4. Build and run the project on your device or in the Android emulator.
 
 The example application ships with a sample publishable key, but if you want to test with your own Stripe account, you can [replace the value of PUBLISHABLE_KEY in PaymentActivity with your test key](https://github.com/stripe/stripe-android/blob/master/example/src/main/java/com/stripe/example/activity/PaymentActivity.java#L25).
+
+Three different ways of creating tokens are shown, with all the Stripe-specific logic needed for each separated into the three controllers,
+`AsyncTaskTokenController`, `RxTokenController`, and `IntentServiceTokenController`.
```

Comment:
+1 just linking to the master branch's version of these files.

Otherwise though, yes LGTM!